oracle/src/events/processAMBAffirmationRequests/index.js

```diff
@@ -75,6 +75,7 @@ function processAffirmationRequestsBuilder(config) {
         txToSend.push({
           data,
           gasEstimate,
+          extraGas: 200000,
```

Comment:
Consider introduce a constant for this. My suggestion is `EXTRA_GAS_ABSOLUTE`.

Reply:
Ok, addressed in 99dc7c0